### ISSUE TITLE
[FEATURE] Update dates

### ISSUE THREAD

**Issue by Zander200-glitch** [CCBlueX/LiquidBounce#369]
**Describe your feature request.**
I would find it cool that if you type .config list, is shows the date the config was last updated, and lists the configs that were recently updated further up in the list.

**Comment by superblaubeere27** [comment]
We could make a convention for AutoConfigs that the authors have to update the date when they are updating the auto-configs.

**Comment by Zander200-glitch** [comment]
But how will that be possible with potentioally modified local time?

**Comment by SenkJu** [comment]
> We could make a convention for AutoConfigs that the authors have to update the date when they are updating the auto-configs.

Why not simply use GitHub's API?

**Comment by Zander200-glitch** [comment]
You mean github's api will pull the dates and apply them to the cfgs?

**Comment by benjaminlaske** [comment]
Exactly, you're able to fetch over an GitHub API endpoint the last modification date of a specific file, which then should be parsed and displayed in the client right next to the specific configuration.

For example here are the [README.md](https://api.github.com/repos/CCBlueX/LiquidBounce/commits?path=README.md&page=1&per_page=1) commits fetched over the GitHub API. Simply parse it for the 0.commit.commiter.date and you'll be good to go. I haven't figured out yet how to fetch a whole folder without attacking the API, however I'm sure someone will figure out.

**Comment by ENDER1355** [comment]
good idea, in order to see if the config is outdated or not

**Comment by ChefZander** [comment]
thats the idea yes lol

**Comment by 1zun4secondary** [comment]
Done.

**Comment by ChefZander** [comment]
damn i got a feature added to LB 🔥
